### ISSUE TITLE
Inconsistent minimum versions

### ISSUE THREAD

**Issue by eviltofu** [AlphaWallet/alpha-wallet-ios#3500]
The Xcode project is set to iOS 11.0. The Cocoapods file is set to 12.0. Should we bump the Xcode project to 12.0 as well?

**Comment by hboon** [comment]
Good point. We can't bump the minimum deployment target for Xcode yet though. I do intend to bump it up more (at least 13 actually). But we have white-label projects that still rely on iOS 11. There is work on WalletConnect v2 and their SDK requires at least 12 (or 13, I forgot), so we'll be moving to at least iOS 12 anyway in this 1-2 months.

So let's leave this for now.

**Comment by eviltofu** [comment]
Also, Travis only tests iOS 15, 14. It should technically test all the way down to the minimum level we support.

**Comment by hboon** [comment]
Yes, but that's ok, our users are dominated by iOS 14 and 15.
